2.7.0 not source compatible (MBEDTLS_DHM_RFC5114_MODP_2048_P)

* OS: linux
* mbedtls 2.7.0

While attempting to build Hiawatha 10.7 against mbedtls 2.7.0 it will fail to find one of the symbols.

    /src/hiawatha-10.7/src/tls.c:348:42: error: ‘MBEDTLS_DHM_RFC5114_MODP_2048_P’ undeclared (first use in this function); did you mean ‘MBEDTLS_DHM_RFC5114_MODP_2048_G’?
       mbedtls_ssl_conf_dh_param(*tls_config, MBEDTLS_DHM_RFC5114_MODP_2048_P, MBEDTLS_DHM_RFC5114_MODP_2048_G);

In 2.6.1 this was defined in `include/mbedtls/dhm.h` and appears to have been removed in commit 784de59 and replaced with `MBEDTLS_DHM_RFC5114_MODP_P`.  This appears to be a typo since the companion variable `MBEDTLS_DHM_RFC5114_MODP_2048_G` did not have its name changed.

--- a/include/mbedtls/dhm.h
+++ b/include/mbedtls/dhm.h
@@ -372,7 +372,7 @@ MBEDTLS_DEPRECATED typedef char const * mbedtls_deprecated_constant_t;
  * in <em>RFC-5114: Additional Diffie-Hellman Groups for Use with
  * IETF Standards</em>.
  */
-#define MBEDTLS_DHM_RFC5114_MODP_P                              \
+#define MBEDTLS_DHM_RFC5114_MODP_2048_P                         \
     MBEDTLS_DEPRECATED_STRING_CONSTANT(                         \
         "AD107E1E9123A9D0D660FAA79559C51FA20D64E5683B9FD1"      \
         "B54B1597B61D0A75E6FA141DF95A56DBAF9A3C407BA1DF15"      \

